.proto/withdrawals.proto

```diff
@@ -59,3 +60,8 @@ message WithdrawalExecuteError {
   ErrorCode code = 1;
   string message = 2;
 }
+
+enum FeePayer {
+  SENDER = 0;
+  RECIPIENT = 1;
```

Comment:
Protofiles are updated automatically on release of API client nuget package.
Instead of this it's needed to update https://github.com/swisschain/Sirius.Api.Docs/blob/master/source/includes/v1/_brokerage-api-withdrawals.md
and https://github.com/swisschain/Sirius.Api.Docs/blob/master/source/includes/v2/_brokerage-api-withdrawals.md